src/containers/WindowSideBarButtons.js

```diff
@@ -41,9 +42,27 @@ const mapStateToProps = (state, { windowId }) => ({
   sideBarPanelId: (getCompanionWindowForPosition(state, windowId, 'left') || {}).id,
 });
 
+/** */
+const style = theme => ({
+  tabsFlexContainer: {
+    flexDirection: 'column',
+  },
+  tabsIndicator: {
+    display: 'none',
+  },
+  tab: {
+    minWidth: 'auto',
+    borderRight: '4px solid transparent',
+  },
+  tabSelected: {
+    backgroundColor: theme.palette.secondary.light,
```

Comment:
Let's revisit these colors in a more holistic palette review with @jvine and @ggeisler.